e2e/redux3/spec/helpers/alias.js

```diff
@@ -1,8 +1,8 @@
 const moduleAlias = require('module-alias');
 
 moduleAlias.addAliases({
-  'proto': '../protos/',
-  '@App': 'neo/'
+  'protos': '../protos/',
+  '@App': '../'
```

Comment:
Ok I wasn't sure what you meant about the tests still being from neo. 👌

Reply:
There was the left over file as well in the spec dir that confused me. Turns out it wasn't being used.